src/backend/gporca/libgpos/include/gpos/memory/CCacheFactory.h

```diff
@@ -60,15 +60,13 @@ class CCacheFactory
 	// private dtor
 	~CCacheFactory()
```

Comment:
Destructor has become empty. Let's delete it

Reply:
done